airflow/www/views.py

```diff
@@ -836,6 +839,21 @@ def log(self, session=None):
         dttm = pendulum.parse(execution_date)
         form = DateTimeForm(data={'execution_date': dttm})
         dag = dagbag.get_dag(dag_id)
+        metadata = {}
+        tailing_required = False
+        tail_lines_list = [100, 200, 500, 1000]
```

Comment:
Don't hard-code defaults like this. Instead put it in to default-airflow.cfg.

Reply:
I am using 3 values from config for this feature. A flag(`tail_logs`), default value(`num_lines`) and comma separated string(`tail_lines_list`). Above hard coded value is added to avoid app crashes in the cases where flag is true and `tail_lines_list` variable is not defined.

Reply:
moved these to `default-airflow.cfg`